File: src/main/kotlin/aba/instantAssistants/CheckStationAction.java
package aba.instantAssistants;

import OSPABA.*;
import aba.simulation.*;
import aba.agents.*;

//meta! id="60"
public class CheckStationAction extends Action
{
	public CheckStationAction(int id, Simulation mySim, CommonAgent myAgent)
	{
		super(id, mySim, myAgent);
	}

	@Override
	public void execute(MessageForm message)
	{
	}

	@Override
	public AgentStation myAgent()
	{
		return (AgentStation)super.myAgent();
	}

}

File: src/main/kotlin/aba/instantAssistants/CheckStationAction.kt
package aba.instantAssistants

import OSPABA.*
import aba.simulation.*
import aba.agents.*

//meta! id="60"
class CheckStationAction(id: Int, mySim: Simulation, myAgent: CommonAgent) : Action(id, mySim, myAgent) {

    override fun execute(message: MessageForm) {}

    override fun myAgent(): AgentStation {
        return super.myAgent() as AgentStation
    }

}
